hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/PrisonerSearchService.kt

```diff
@@ -132,6 +133,11 @@ class PrisonerSearchService(
         size(RESULT_HITS_MAX)
         query(it)
       }
+      // TODO: remove the -[0-9]* suffix from the client
+      fields.clients[authenticationHolder.clientId]?.apply {
```

Comment:
eventually would expect that an exception is thrown if client fields are not defined for the given client id